test/e2e/alertmanager_instance_namespaces_test.go

```diff
@@ -28,6 +28,7 @@ import (
 
 func testAlertmanagerInstanceNamespacesAllNs(t *testing.T) {
 	ctx := framework.NewTestCtx(t)
+
```

Comment:
nit: there is no need for a new line here, usually we tend to put an object creation and its deferred deletion next to each other. It helps not forgetting about the cleanup.

Reply:
Fixed :)

Reply:
The test that was passing before failing after the change. To make sure its not because of new commit I will try removing it once and add back

Reply:
It was still failing so I added back the commit

Reply:
Understood the error now I removed the extra spaces in func calls. I was not aware of that looks like my IDE added those :|. Generate step is passed now